lib/particles/computed_particle.dart

```diff
@@ -17,9 +15,10 @@ class ComputedParticle extends Particle {
   ParticleRenderDelegate renderer;
 
   ComputedParticle({
-    @required this.renderer,
-    double lifespan,
-    Duration duration,
+    required this.renderer,
+    double? lifespan,
+    /// FIXME: why `duration` is here?
+    Duration? duration,
```

Comment:
@luanpotter I didn't get why `duration` is here at all

Reply:
cc @spydon

Reply:
sorry I confused particles with effects haha. I will take a look, maybe @av knows?

Reply:
Removed for good